stone/lang/lexer.py

```diff
@@ -100,41 +102,31 @@ def test(self, data):
         'KEYWORD',
         'PATH',
         'DOT',
-    )
 
-    # Whitespace tokens
-    tokens += (
+        # Whitespace tokens
```

Comment:
I'm more just wondering how smart mypy is. Could you just have typed the first block as `# type: Tuple[str, ...]` and left all concatenations untouched?